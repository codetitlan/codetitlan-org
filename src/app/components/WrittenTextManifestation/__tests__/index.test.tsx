import * as React from 'react';
import { render } from '@testing-library/react';

import { WrittenTextManifestation } from '..';

const mockDialogue = ['a', 'b'];

describe('<WrittenTextManifestation  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <WrittenTextManifestation dialogue={mockDialogue} />,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });

  it('should call a function on end', () => {
    const loadingIndicator = render(
      <WrittenTextManifestation dialogue={mockDialogue} onDone={() => []} />,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
