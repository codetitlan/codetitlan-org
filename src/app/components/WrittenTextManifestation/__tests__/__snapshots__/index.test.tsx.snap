// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WrittenTextManifestation  /> should call a function on end 1`] = `
<div
  class="Typewriter"
  data-testid="typewriter-wrapper"
>
  <span
    class="Typewriter__wrapper"
  />
  <span
    class="Typewriter__cursor"
  >
    |
  </span>
</div>
`;

exports[`<WrittenTextManifestation  /> should match snapshot 1`] = `
<div
  class="Typewriter"
  data-testid="typewriter-wrapper"
>
  <span
    class="Typewriter__wrapper"
  />
  <span
    class="Typewriter__cursor"
  >
    |
  </span>
</div>
`;
